use-timescale/metrics-logging/metrics-to-prometheus.md

```diff
@@ -10,65 +10,102 @@ cloud_ui:
 
 # Export metrics to Prometheus
 
-You can export your Timescale service telemetry to Prometheus.
+[Prometheus][prometheus] is an open-source monitoring system with a dimensional data model, flexible query language, and a modern alerting approach.
 
-## Create a monitoring user
+This page shows you how to export your $SERVICE_SHORT telemetry to Prometheus.
 
-1. Connect to the Timescale service as tsdbadmin (using psql or a similar tool).
+## Prerequisites
 
-1. Create a new user to scrape the metrics from the database using the required permissions:
+<IntegrationPrereqs />
 
-    a. Create a user named `monitoring` with the desired password by replacing `<password>`:
-    
-    ```sql
-    CREATE USER monitoring WITH PASSWORD '<password>';
-    ```
+- [Download and install Prometheus][install-prometheus].
 
-    b. Grant `pg_read_all_stats` to the `monitoring` user:
+## Export $SERVICE_SHORT telemetry to Prometheus
 
-    ```sql
-    GRANT pg_read_all_stats to monitoring;
-    ```
+Take the following steps to export your data:
 
-## Install PostgreSQL exporter
+<Procedure>
 
-1. Install the postgresql_exporter on a host that you manage to connect to the Timescale service and collect telemetry. Below is an example using Docker to run postgresql_exporter:
+1. **Connect to your $SERVICE_LONG as a privileged user**
 
-    make sure to replace `<PASSWORD>` with the created password and `<TIMESCALE-HOST:PORT>` with the timescale service host and port.
+   See the available [connection options][run-queries]. For self-hosted installations, use [`psql`][psql].
 
-    ```bash
-    docker run \
-    -p 9187:9187 \
-    -e DATA_SOURCE_NAME="postgres://monitoring:<PASSWORD>@<TIMESCALE-HOST:PORT>/tsdb?sslmode=require" \
-    quay.io/prometheuscommunity/postgres-exporter
-    ```
-    To reduce latency and potential data transfer costs,  best practice is to run `postgresql_exporter` in the same AWS region as your Timescale Cloud service.
+1. **Create a user to scrape the metrics**
+
+   1. Create a user named `monitoring` with a password:
+
+      ```sql
+      CREATE USER monitoring WITH PASSWORD '<password>';
+      ```
+
+   1. Grant the `pg_read_all_stats` permission to the `monitoring` user:
+
+      ```sql
+      GRANT pg_read_all_stats to monitoring;
+      ```
+
+1. **Install PostgreSQL Exporter**
 
-1. Once the postgresql_exporter is up and running, and successfully connected to the Timescale service, you can configure your Prometheus server to scrape the postgresql_exporter metrics endpoint. This endpoint exposes all the metrics provided by the exporter. 
+   PostgreSQL Exporter collects PostgreSQL performance metrics and exposes them in a Prometheus-compatible format. [Install PostgreSQL Exporter][install-exporter] on the host that you use to connect to your $SERVICE_SHORT and collect telemetry. To reduce latency and potential data transfer costs, run PostgreSQL Exporter in the same AWS region as your $SERVICE_LONG.
```

Comment:
```suggestion
   You Install PostgreSQL Exporter on the machine that you use to connect to your $SERVICE_SHORT and collect telemetry. To reduce latency and potential data transfer costs, run PostgreSQL Exporter in the same AWS region as your $SERVICE_LONG.
```

You don't need the link as you explain how to install next.

Reply:
I don't fully explain how to install (there are many options and many platforms and we decided not to provide those steps in our integration pages), but only provide an example where you need to use connection details.

Reply:
Moved to the prerequisistes